src/site/content/en/react/how-to-use-amp-in-nextjs-and-why/index.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: post
+title: How to use AMP in Next.js (and why)
+authors:
+  - houssein
+subhead: |
+  Add AMP to any of your Next.js pages to guarantee fast page loads
+date: 2019-10-17
+draft: true
+---
```

Comment:
The page is now ~1250 words. We usually try to cap pages at ~1000 words. I'll do a pass later to see if anything can be removed, but otherwise I'm fine with it being a little long.